frontend/Screens/Home.tsx

```diff
@@ -57,10 +55,9 @@ const Home = ({navigation}) => {
     // }, [])
 
     return <View>
-        <Text>you're logged in</Text>
+        <Text>you're logged in!</Text>
         <Button onPress={() => logOut()} title={'logOut'}/>
```

Comment:
this button is almost invisible on an iphone device. It is also no longer clickable due to the notch that an iPhone has

Reply:
send screenshot